presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1404,6 +1414,34 @@ void PrestoServer::reportNodeStatus(proxygen::ResponseHandler* downstream) {
   http::sendOkResponse(downstream, json(fetchNodeStatus()));
 }
 
+void PrestoServer::handleGracefulShutdown(const std::vector<std::unique_ptr<folly::IOBuf>>& body , proxygen::ResponseHandler* downstream){
+    std::string requestBody;
+    for (const auto& buf : body) {
+        // Convert IOBuf to fbstring and then to std::string
+        requestBody += buf->moveToFbString().toStdString();
+    }
+    requestBody.erase(requestBody.length()-2);
```

Comment:
Please add a comment describing this erase.

Reply:
Done.